src/controllers/settings.js

```diff
@@ -3,7 +3,36 @@ import { settings } from '../views/partials/settings.js'
 
 function settingsPage (req, res) {
   const data = {
-    partial: settings
+    partial: settings,
+    // TODO: The entries below are for testing only and should be removed.
```

Comment:
This data will be added by PR #2743 which introduces the backend changes for the settings page.